### DISCUSSION TITLE
How to set none facialHair for Personas style

### DISCUSSION THREAD

**Discussion by marekmez** [dicebear/dicebear#218] [Q&A] [answered]
Hi, I was trying to make some female avatars, but unfortunately, I am not able to set faicialHair to `none` even though the Persoans builder allows for it.

![Screenshot 2022-05-22 104741](https://user-images.githubusercontent.com/18588122/169704047-e7932282-9e61-49c9-b98c-0886ea394447.jpg)

Any idea how to set it up? I am using HTTP API. Thanks

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @marekmez 

You can pass an empty array to the API as follows:
https://avatars.dicebear.com/api/personas/your-custom-seed.svg?hair[]

This way the API will not have any hair styles to choose from and the hair will not be present.

**Reply by marekmez** [reply]
@FlorianKoerner that works, thank you for the quick reply. Btw amazing project!
